### PR TITLE
DRAFT - TMEDIA-1 - Chromatic setup

### PR DESCRIPTION
**Pull request by matthewroach**
**Do not approve any chromatic tests**

Chromatic project - 
https://www.chromatic.com/builds?appId=5f91e4721ccaba0022a10782

Storybook for this branch
https://5f91e4721ccaba0022a10782-xucizqqoha.chromatic.com/?path=/story/intro--page

### CODE REVIEW

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# .github/workflows/chromatic.yml
+
+name: 'Chromatic'
+
+# Event for the workflow
+on:
+  pull_request:
+    types: [synchronize, opened]
+    branches-ignore:
+      - "dependabot/**"
```
Comment (JackHowa) [inline .github/workflows/chromatic.yml:10]: nice

File: .github/workflows/chromatic.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+jobs:
+  chromatic-deployment:
+    # Operating System
+    runs-on: ubuntu-latest
+    # Job steps
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # 👈 Required to retrieve git history
```
Comment (JackHowa) [inline .github/workflows/chromatic.yml:22]: 👍

File: blocks/share-bar-block/features/share-bar/default.jsx [hunk 3]
```diff
@@ -97,7 +100,7 @@ const ShareBarContainer = () => {
 };
 
 export const ShareBar = ({
-  customFields, websiteName, websiteDomain, websiteUrl, headlineString,
+  customFields = {}, websiteName, websiteDomain, websiteUrl, headlineString,
```
Comment (JackHowa) [inline blocks/share-bar-block/features/share-bar/default.jsx:103]: is this new?

**Review by JackHowa — DISMISSED** [review]
Looks good to me. Any documentation for someone who is new to storybook would probably be helpful. Otherwise, nice work. Good job ignoring dependabot as I've seen its downsides.

File: blocks/share-bar-block/features/share-bar/default.jsx (hunk 3 above)
Comment (matthewroach) [reply blocks/share-bar-block/features/share-bar/default.jsx:103]: Yes, a small update to fix and existing issue where the share bar renders with a blank div with box shadow

Comment (matthewroach) [reply blocks/share-bar-block/features/share-bar/default.jsx:103]: Also updated as it's a small component which allows me to use as reference, but issue was it had this issue I needed to fix

**Pull request merged**